python/carts.py

```diff
@@ -0,0 +1,34 @@
+import datetime
+import sdk.python.entities
+from sdk.python.entities import Property
+from sdk.python.users import User
+from sdk.python.domains import Domain
+from sdk.python.shipments import Shipment
+from sdk.python.companies import Company
+from sdk.python.cart_items import CartItem
+
+
+class Cart(sdk.python.entities.Entity):
+
+    resource = '/carts/'
+    json_name = 'cart'
+
+    creation_date = Property(datetime.datetime)
+    id = Property(int)
+    ip = Property(str)
+    token = Property(str)
+    cost = Property(float)
```

Comment:
@Amtron do you mind update attributes in this pr, since now you have subtotal_cost, tax_amount, total_cost, items_subtotal_cost, items_tax_amount, items_total_cost